CIP-????/spec.cddl

```diff
@@ -0,0 +1,21 @@
+; CIP-XXXX: CIP-Descriptor
```

Comment:
@Crypto2099 can clarify this better, but @BrockCruess I believe these schemas should be path-named and formatted as per CIP-0088 as for the other token CIPs.  We won't have a final directory name without a CIP number but I think it wouldn't be too soon to name & construct these files here as documented on the landing page: https://github.com/cardano-foundation/CIPs/tree/master/CIP-0088/CIPs

Reply:
@Crypto2099 @Ryun1 @KtorZ I'm also thinking now it might make sense to have a CIP-0088 label like we have for these other registry extensions?
- https://github.com/cardano-foundation/CIPs/pulls?q=label%3A%22CIP-0010%3A+new+registry+entry%22
- https://github.com/cardano-foundation/CIPs/pulls?q=label%3A%22CIP-0067%3A+new+label%22

Reply:
That will probably make sense as I imagine that CIP-88 or CIP-72 style "authentication" standards (as well as the one that we recently approved for CBOR labels) will probably become more prevalent over time.

Reply:
@Crypto2099 @Ryun1 here it is then 🤓
- https://github.com/cardano-foundation/CIPs/labels/CIP-0088%3A%20new%20extension

Reply:
@BrockCruess I'm resolving my initial request above since @Crypto2099 in today's CIP meeting today has pointed out that inclusion in the CIP-0088 document structure is intended for standards of sufficient stability and therefore wouldn't be relevant until this proposal is merged & later is assigned `Active` status... so isn't a requirement for being merged as `Proposed`.